api/pkg/service/rating/rating.go

```diff
@@ -50,10 +51,21 @@ func New(api app.Config) rating.Service {
 // Find user's rating for a resource
 func (s *service) Get(ctx context.Context, p *rating.GetPayload) (*rating.GetResult, error) {
 
+	// Required scope to get the user rating for a resource
+	requiredScope := &security.JWTScheme{
+		RequiredScopes: []string{"rating:read"},
+	}
+
+	//Verify and validate the required scopes
+	newctx, err := s.JWTAuth(ctx, p.Session, requiredScope)
+	if err != nil {
+		return &rating.GetResult{Rating: 0}, err
+	}
+
 	req := request{
-		db:     s.DB(ctx),
-		log:    s.Logger(ctx),
-		userID: validator.UserID(ctx),
+		db:     s.DB(newctx),
```

Comment:
Why can't we use the same context ?

Reply:
It was not adding userID in the context while doing jwtAuth validation